### ISSUE TITLE
how to fix it A0011001, A0011003

### ISSUE THREAD

**Issue by edh-git** [edh-git/EL_Display_Hub#1723]
Product:EasyCanvas<br><br>Device OS:iOS<br><br>e-mail:chgikem@gmail.com<br><br>i'm reboot not working<br><br>[EL_Display_50029eae-3196-4000-8b76-9701db0d8314.zip](https://github.com/edh-git/EL_Display_Hub/blob/main/EL_Display_50029eae-3196-4000-8b76-9701db0d8314.zip)

**Comment by james-kim-Devguru** [comment]
해당 사용자에게 아래와 같이 회신하였습니다.

---
#### 제목
```
Re: [Twomon SE - 1723] how to fix it A0011001, A0011003
```
#### 본문
```
Hello, customer. This is Easy&Light support team.

First of all, we apologize for any inconvenience caused.

As a result of checking your symptoms, it seems that the program was not installed normally.
Please proceed with reinstalling the PC program.

1. Delete "EL Display Hub"
2. Reboot PC (Use "Restart" in Shutdown option)
3. Download the new installer and reinstall "EL Display Hub"
[EL Display Hub] installation page: http://www.easynlight.com/en/easycanvas/

If the problem is not improved even after taking the above measures, please refer to the Log Acquisition Guide and attach the PC log and application log and ask again.

[FAQ] Log Acquisition Guide: http://www.easynlight.com/en/supporteasycanvas/?faq=54

Once again, we apologize for any inconvenience caused.
We will try to make better products.

Thank you

Easy&Light
```

**Comment by RachelOh** [comment]
2023.4.15. 오후 9:25
---
Email : [chgikem@gmail.com](mailto:chgikem@gmail.com)
Device OS: iOS
Product: EasyCanvas

The problem can't be solved. I have rebooted and reinstalled dozens of times. I haven't been able to use this program for months. Help find new solutions that are more than just Delete and install again

![Screenshot 2023-04-15 192207](https://user-images.githubusercontent.com/77039864/232354525-5e2f8f01-5d42-4f38-89fe-e87d0865b260.png)
[EL_Display_5c73664f-d375-4102-b506-a52345c88096.zip](https://github.com/edh-git/EL_Display_Hub/files/11245223/EL_Display_5c73664f-d375-4102-b506-a52345c88096.zip)

**Comment by james-kim-Devguru** [comment]
해당 사용자에게 아래와 같이 회신하였습니다.

---
#### 제목
```
Re: [EasyCanvas - 1723] how to fix it A0011001, A0011003
```
#### 본문
```
Hello, customer. This is Easy&Light support team.

I reviewed the reinstallation log and it says driver installation is failing.
We will guide you on how to resolve the issue.

These are common fixes to fix driver errors.
1. Remove the EL Display Hub.
2. Update to the latest Windows.
3. Download the new installer and reinstall "EL Display Hub"
     [EL Display Hub] installation page: http://www.easynlight.com/en/easycanvas/

If the problem is not resolved, try manually deleting the driver.
1. Remove the EL Display Hub.
2. Remove the driver package file of EL Display Hub.
     * If an error occurs, move the file to another folder without removing it for restoration.
     * Move the following files to another folder.
         * "C:\Windows\INF\oem28.inf"
         * "C:\Windows\INF\oem28.PNF"
3. Reboot your PC.
4. Download the new installer and reinstall "EL Display Hub"
     [EL Display Hub] installation page: http://www.easynlight.com/en/easycanvas/

If the problem does not improve even after taking the above measures, please attach a new PC log and application log and contact us again.

[FAQ] Log Acquisition Guide: http://www.easynlight.com/en/supporteasycanvas/?faq=54

We will always strive for better products and services.
Thank you

Easy&Light
```
